internal/cli/opsmanager/agents/upgrade.go

```diff
@@ -29,6 +29,10 @@ type UpgradeOpts struct {
 	store store.AgentUpgrader
 }
 
+var upgradeTemplate = `AGENT VERSION	BI VERSION
+{{.AutomationAgentVersion}}	{{.BiConnectorVersion}}
+`
```

Comment:
[nit] for create/update or any other action like that we are simplifying to be something like what we do with deletes
so this
```suggestion
var upgradeTemplate = "Updating agent versions to the latest available."
```
Or something like that, you could also throw the new version there if you want but that method always go to latests